include/nuttx/mm/iob.h

```diff
@@ -114,6 +114,14 @@ struct iob_s
 #endif
   unsigned int io_pktlen; /* Total length of the packet */
 
+#ifdef CONFIG_IOB_SHARED
+  /* Pointer to parent node */
+
+  FAR struct iob_s *io_parent;
+
+  unsigned int io_refs; /* Reference count of this iob */
```

Comment:
why the reference count not part of io_data?

Reply:
Why need part of io_data ?